Lib/tkinter/test/test_tkinter/test_font.py

```diff
@@ -100,6 +100,10 @@ def test_names(self):
             self.assertIsInstance(name, str)
             self.assertTrue(name)
         self.assertIn(fontname, names)
+    
+    def test_repr(self):
+        self.assertEqual(repr(self.font), f'<{self.font.__class__.__module__}.{self.font.__class__.__qualname__} object {fontname!r}>')
```

Comment:
Too long line.

And I think it would be enough to hardcode the class name: `tkinter.font.Font`.